src/sidebar/directive/annotation.js

```diff
@@ -70,6 +77,20 @@ function AnnotationController(
           savedAnnot[k] = annot[k];
         }
       });
+
+
+      if(vm.isReply()){
```

Comment:
In future I'd like us to have tests for this, but testing all of the cases here is going to be a PITA for the moment given that this code is not separated from the rest of the UI.